docs/control-structures.rst

```diff
@@ -398,17 +398,17 @@ If used properly, analysis tools can evaluate your contract to identify the cond
 There are two other ways to trigger exceptions: The ``revert`` function can be used to flag an error and
 revert the current call. It is possible to provide a string message containing details about the error
 that will be passed back to the caller.
-The deprecated keyword ``throw`` can also be used as an alternative to ``revert()`` (but only without error message).
 
 .. note::
-    From version 0.4.13 the ``throw`` keyword is deprecated and will be phased out in the future.
+    There used to be a keyword called ``throw`` with the same semantics as ``revert()`` which
+    whas deprecated in version 0.4.13 and removed in version 0.5.0.
 
 When exceptions happen in a sub-call, they "bubble up" (i.e. exceptions are rethrown) automatically. Exceptions to this rule are ``send``
-and the low-level functions ``call``, ``delegatecall``, ``callcode`` and ``staticcall`` -- those return ``false`` in case
+and the low-level functions ``call``, ``delegatecall`` and ``staticcall`` -- those return ``false`` in case
```

Comment:
Maybe worth mentioning that it returns false in the `success` part of the return values

Reply:
Good catch!